src/main/java/seedu/address/logic/commands/CreateCommand.java

```diff
@@ -27,8 +28,8 @@
             + PREFIX_USERNAME + "username "
             + PREFIX_PASSWORD + "password ";
 
-    public static final String MESSAGE_SUCCESS = "New user added: %1$s";
-    public static final String MESSAGE_FAILURE = "User already exist!";
+    private static final String MESSAGE_SUCCESS = "New user added successfully!";
+    private static final String MESSAGE_FAILURE = "Username already exist.";
```

Comment:
is username already exists the only possible source for failure for create command?